resources/views/main/survey.blade.php

```diff
@@ -0,0 +1,17 @@
+@extends('layouts.app')
+
+@section('content')
+
+
+<h1>Survey blade</h1>
+
+<h1>Tilgjengelige innstillinger</h1>
```

Comment:
Jeg må se det litt nøyere senere, men mulig alle h1 burde være h2 pga embedding på side med etabliert h1. Wcag 2.1

Reply:
@theahthodesen

Reply:
@manilpit Denne fila er bare en placeholder foreløpig, den blir oppdatert når Audun merger inn 738. Per nå er den kun for å bli tatt til "riktig" fil når man setter inn survey, men selve survey-lti vinduet har ikke noen funksjon før 738 blir merget inn.